CHANGELOG.md

```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
 
 # Unreleased
 
+## Sprint 27 end 2015-08-03
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+27+Ending+August+3%2C+2015%22+label%3Aenhancement)
+
+- Providers: Pluggable provider namespacing
+- Self Service  
+ - Project Jellyfish (Booz Allen Hamilton - Founding Community Partner)
+ - /spa\_ui/self_service
+- Automate: Multiple state machine support
+- Provisioning: Multi-tenant support for Orchestration provisioning in Openstack
+- Rest API
+ - Chargebacks & rates queries
+ - Reports & results queries
+- Appliance
+ - LinuxAdmin: Added systemctl support and automatic service management tool 
+ - Support for PostgreSQL 9.4.1
+
+
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+27+Ending+August+3%2C+2015%22+label%3Aenhancement)
+
+- Providers: SCVMM Inventory Performance Improvement
+- UI
+ - Replaced DHTMLXCombo controls
+ - Replaced DHTMLXCalendar controls
+ - Updates for Container support
+- Rest API: Source refactoring
+ - app/helpers/api\_helper/ → app/controllers/api_controller
+- Appliance: Replication resiliency
```

Comment:
Instead of `Appliance: Replication` let's just say `Replication` and we can probably move resiliency to the next line.  `- Added heartbeating of child worker process for resiliency`